### ISSUE TITLE
https://m.gsmarena.com/realme_gt_neo2-11074.php

### ISSUE THREAD

**Issue by K11MCH1** [skyline-emu/skyline-games-list#267]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

1078

### Android Version

11

### SoC type

Snapdragon

### Device GSMArena link

https://m.gsmarena.com/realme_gt_neo2-11074.php

### Game Behaviour

- Insta crash.

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

[The Witcher 3.zip](https://github.com/skyline-emu/skyline-games-list/files/8639785/The.Witcher.3.zip)

**Comment by EZonTheEyes** [comment]
Reaching In-Game on Edge 27 (HUD only)
![image](https://user-images.githubusercontent.com/103911396/213888922-ba8e64db-2819-4aee-ab95-178c6972dd3b.png)
![image](https://user-images.githubusercontent.com/103911396/213888967-0c9fd9a4-a7e2-418d-a851-9a32cf40dc14.png)

**Comment by Crytonics** [comment]
> Reaching In-Game on Edge 27 (HUD only) ![image](https://user-images.githubusercontent.com/103911396/213888922-ba8e64db-2819-4aee-ab95-178c6972dd3b.png) ![image](https://user-images.githubusercontent.com/103911396/213888967-0c9fd9a4-a7e2-418d-a851-9a32cf40dc14.png)

Can you upload log?

**Comment by EZonTheEyes** [comment]
> > Reaching In-Game on Edge 27 (HUD only) ![image](https://user-images.githubusercontent.com/103911396/213888922-ba8e64db-2819-4aee-ab95-178c6972dd3b.png) ![image](https://user-images.githubusercontent.com/103911396/213888967-0c9fd9a4-a7e2-418d-a851-9a32cf40dc14.png)
> 
> Can you upload log?

yep, redumping game soon to a device that can test it with mesa drivers aswell. this was one of the few instances I forgot

**Comment by Synz123** [comment]
The game now actually "renders" on Edge 69 (Using SD855 with mesa3d turnip drivers) but horribly, the frame rate is low even with a overclocked sd855
![Screenshot_20230508-104754_Skyline Edge](https://user-images.githubusercontent.com/87271713/236767249-7735a31d-91ac-454c-9464-77b10031218e.png)
